Gopkg.toml

```diff
@@ -26,7 +26,7 @@
 
 [[constraint]]
   name = "github.com/Scalingo/go-scalingo"
-  branch = "master"
+  branch = "fix/cli/398/addon_logs"
```

Comment:
https://github.com/Scalingo/go-scalingo/pull/78/files being merged, you can now use master.